webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -0,0 +1,171 @@
+
+  useEffect(() => {
+    document.body.classList.add('transprenet-border');
+  }, []);
+
+  const handleTabClick = (event, tabIndex) => {
+    setActiveTab(tabIndex);
+  };
+  return (
+    <>
+      <PageSection className="header" variant={PageSectionVariants.light}>
+        <div style={{ marginLeft: '18px', marginRight: '18px' }}>
+          <Breadcrumb style={{ marginTop: '15px' }}>
+            <BreadcrumbItem to="/hosts">Hosts</BreadcrumbItem>
+            <BreadcrumbItem isActive>
+              {response.name || <Skeleton />}
+            </BreadcrumbItem>
+          </Breadcrumb>
+          {/* TODO: Replace all br with css */}
+          <br />
+          <br />
+          <Grid>
+            <GridItem span={2}>
+              <Title headingLevel="h5" size="2xl">
+                {/* TODO: Make a generic Skeleton HOC (withSkeleton) */}
+                {response.name || <Skeleton />}
+              </Title>
+            </GridItem>
+            <GridItem style={{ marginTop: '5px', marginLeft: '10px' }} span={8}>
+              <Badge key={1}>{response.operatingsystem_name}</Badge>{' '}
+              <Badge key={21}>{response.architecture_name}</Badge>
+            </GridItem>
+            <GridItem span={2}>
+              <ActionsBar hostName={response.name || <Skeleton />} />
+            </GridItem>
+          </Grid>
+          <Text style={{ fontStyle: 'italic' }} component={TextVariants.p}>
+            {/* TODO: extracting text and remove timeago usage in favor i18n */}
+            {response.name ? (
+              <div>
+                created{' '}
+                <RelativeDateTime
+                  date={response.created_at}
+                  defaultValue="N/A"
+                />{' '}
+                by {response.owner_name} (updated{' '}
+                <RelativeDateTime
+                  date={response.updated_at}
+                  defaultValue="N/A"
+                />
+                )
+              </div>
+            ) : (
+              <Skeleton width={400} />
+            )}
+          </Text>
+          <br />
+        </div>
+      </PageSection>
+      <Tabs
+        style={{
+          marginLeft: '-18px',
+          marginRight: '-18px',
+          background: 'white',
+        }}
+        activeKey={activeTab}
+        onSelect={handleTabClick}
+      >
+        <Tab eventKey={0} title="Details">
+          <div id="test">
+            <br />
+            <Grid>
+              <GridItem offset={3} span={4}>
+                <StatusAlert
+                  status={response ? response.global_status_label : null}
+                />
+              </GridItem>
+            </Grid>
+            <br />
+            <br />
+            <Grid>
+              <GridItem span={3} rowSpan={3}>
+                <Properties hostData={response} />
+              </GridItem>
+              <GridItem style={{ marginLeft: '40px' }} span={3}>
+                <ParametersCard paramters={response.all_parameters} />
```

Comment:
What's going to be in this parameters section? didn't see it on the original page
![Selection_338](https://user-images.githubusercontent.com/26363699/87458298-71d95180-c612-11ea-86bc-a9fa4e19d352.png)

Reply:
Parameters that use by this host, but the list can be big, I wonder to move this section as a different tab with a dedicated table. in the `details` tab we can have a high-level overview such as - last audits, last reports, last tasks last updated facts(?) and etc.